doc/lightning-openchannel_signed.7.md

```diff
@@ -0,0 +1,65 @@
+lightning-openchannel\_signed -- Command to conclude a channel open
+===================================================================
+
+SYNOPSIS
+--------
+
+**openchannel_signed** *id* *signed_psbt* \[*timeout*\]
+
+DESCRIPTION
+-----------
+
+`openchannel_signed` is a low level RPC command which concludes a channel
+open with the specified peer. It uses the v2 openchannel protocol, which
+allows for interactive transaction construction.
+
+This command should be called after a call to `openchannel_update`
+with a return value that includes a true *commitments_secured*.
+
```

Comment:
IOW, "should be called after `openchannel_update` returns *commitments_secured* `true`."